client/my-sites/theme/main.jsx

```diff
@@ -922,6 +927,13 @@ class ThemeSheet extends Component {
 
 		return (
 			<Main className={ className }>
+				{ siteSlug && (
```

Comment:
Why do we display the global notices only when there's a site slug? What's the rationale? How would this make sense in a year from now when we want to render a notice on this page and we don't care about the current site and if there's even a current site?

Reply:
I wanted to limit the notices only for logged-out screens, but I guess it makes sense to make it available for all screens.